gnd/src/main/java/com/google/android/gnd/repository/TermsOfServiceRepository.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.repository
+
+import com.google.android.gnd.model.TermsOfService
+import com.google.android.gnd.persistence.local.LocalValueStore
+import com.google.android.gnd.persistence.remote.RemoteDataStore
+import com.google.android.gnd.rx.annotations.Cold
+import io.reactivex.Maybe
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+
+private const val LOAD_REMOTE_PROJECT_TERMS_OF_SERVICE_TIMEOUT_SECS: Long = 30
+
+class TermsOfServiceRepository @Inject constructor(
+    private val remoteDataStore: RemoteDataStore, private val localValueStore: LocalValueStore) {
+
+    val termsOfService: @Cold Maybe<TermsOfService>
```

Comment:
you can hide the setter for this if we don't call it elsewhere:

`private set`

Reply:
oops my bad, this is a val so there is no setter